platforms.md

```diff
@@ -0,0 +1,52 @@
+# Mantle platforms
+
+Platforms are a stable API interface to different environments to run clusters,
+create images, collect logging information, etc.
+
+## Auth
+
+Auth differs based on the platform. Some platforms like `aws` utilize the
+configuration files from their command-line tooling while others define their
+own custom configuration format and default locations (like [DigitalOcean](https://github.com/coreos/mantle/blob/v0.9.1/auth/do.go)).
+Generally if any extensions / custom configurations are needed a new file is
+created inside of the `auth` package which will define the default location,
+the structure of the configuration, and a function to parse the configuration
+file (usually named Read<platform>Config and emits a
+`map[string]<platform>Config` object).
+
+## API
+
+Platform APIs wrap the golang SDK and live inside of `platform/api/<platform>/`.
+There is no direct requirement for what functionality is present in the API.
+
+## Cluster & Machine
+
+Clusters must implement this [interface](https://github.com/coreos/mantle/blob/v0.9.1/platform/platform.go#L75-L97).
```

Comment:
s/this/the `Cluster` interface
Perhaps a little obvious but if the line numbers change the name wont go stale

Reply:
Also why linking to the tag?